x-pack/legacy/plugins/uptime/public/lib/helper/__tests__/__snapshots__/parameterize_values.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`parameterizeValues parameterizes provided values for multiple fields 1`] = `"&foo=bar&foo=baz&bar=foo&bar=baz"`;
```

Comment:
No, because this function is intended to be a generic solution. The implementation details of the code that resulted in its creation had required parameters that will always be present, but this might not always be the case. It's the responsibility of the caller to format its request params appropriately. 👍 https://github.com/elastic/kibana/pull/50837/commits/22b34f4eea29363457e4cff69351ebae3bca5fa9